test/schema_test.ts

```diff
@@ -278,7 +309,13 @@ describe('Schema', () => {
           fromKey: 'another',
           properties: {
             Boo: {type: schema.ValueType.String, fromKey: 'boo'},
-            Baz: {type: schema.ValueType.Number, fromKey: 'baz', mutable: true, autocomplete: AutocompleteType.Dynamic},
+            Baz: {
+              type: schema.ValueType.String,
```

Comment:
Changed this test to use to a `ValueType.String` since `Number` doesn't support `options`